MAUI/PDF-Viewer/Printing-PDF-files.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: Printing PDF files in .NET MAUI PDF Viewer control | Syncfusion
+description: Learn here all about printing feature in Syncfusion .NET MAUI PDF Viewer (SfPdfViewer) control and more.
+platform: MAUI
+control: SfPdfViewer
+documentation: ug
+---
+
+# Printing PDF Files in .NET MAUI PDF Viewer (SfPdfViewer)
+
+The print feature of [SfPdfViewer](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html) allows you to effortlessly print PDF documents directly from your application code. Also, the modified form fields and annotations in SfPdfViewer will be preserved properly in the printed document.
+
+![Printing PDF Files in .NET MAUI PDF Viewer](Images\Print\print.jpg)
```

Comment:
* Image can be placed at the end of the content (before the limitations).
* Image description sentence is missing. "The following image represents...."

Reply:
Changed the position of the image .
Added the image description .